network/static/js/observation_view.js

```diff
@@ -51,16 +51,37 @@ $(document).ready(function() {
         var wavesurfer = Object.create(WaveSurfer);
         var data_payload_url = $this.data('payload');
         var container_el = '#data-' + wid;
+        $(container_el).css('opacity', '0');
         var loading = '#loading-' + wid;
         var $playbackTime = $('#playback-time-' + wid);
+        var progressDiv = document.querySelector('#progress-bar-' + wid);
+        var progressBar = progressDiv.querySelector('.progress-bar');
```

Comment:
Most of the code was part of a similar example, that's why I used document.queryselector.

As you proposed, I've changed it.